classes/player/player.gd

```diff
@@ -178,8 +178,11 @@ onready var feet_area: Area2D = $Feet
 
 
 func _ready():
+	
 	switch_state(S.NEUTRAL) # reset state to avoid short mario glitch
 	
+	Singleton.reset_bus_effect("~Water Verb:", 0) # prevents garbage reverb
```

Comment:
should this be called from the player script? i feel like this adds unnecessary code to the player and could be made more robust by calling from somewhere else
probably somewhere that'll directly link it to scene transitions

Reply:
That would probably help, but I would relegate that to its own PR. I think this is fine for 0.1.6.